docs/_docs/07-dg-code-structure.md

```diff
@@ -6,3 +6,28 @@ last_modified_at: 2020-06-20
 toc: true
 ---
 
+Hyperspace code is written in Scala and it is using 'sbt' as the build tool. Below is the list of folders in the code base hierarchy:
+### docs
+“docs” folder should be the starting point for looking into Hyperspace code base. It contains useful documentations on different aspects of Hyperspace project, including coding and formatting guidelines, Hyperspace configurations and information on how to build the project in various environments. 
+### src
+“src” folder contains Hyperspace’s source and test code. 
+The source code is available under:
+* src/main/scala/com/microsoft/hyperspace
+
+The source code includes implementation of core functionality and features of Hyperspace and is organized in different packages:
+- **actions** : It includes the implementation of Hyperspace actions to create, maintain and remove indexes in a concurrent manner.
+- **index**: It includes implementation of Hyeprspace indexes, optimizations rules and other core features. 
+- **util**: Various utility codes that are used by different features and components are added here.
+
```

Comment:
how about code at `src/main/scala/com/microsoft/hyperspace`? and `org/apache/spark`?

Reply:
Done

Reply:
How about something like:

**`src/main/scala/com/microsoft/hyperspace`**
&nbsp;&nbsp;&nbsp;|- **`actions`**: blah
&nbsp;&nbsp;&nbsp;|- **`index`**: blah
**`src/main/scala/org/apache/spark`**
&nbsp;&nbsp;&nbsp;|- **`dir`**: blah

Reply:
Done.